.github/workflows/docs-preview.yaml

```diff
@@ -17,89 +17,26 @@ permissions:
   security-events: none
   statuses: none
 
-# Cancel in-progress runs for pull requests when developers push
-# additional changes, and serialize builds in branches.
-# https://docs.github.com/en/actions/using-jobs/using-concurrency#example-using-concurrency-to-cancel-any-in-progress-job-or-run
-concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
+
 
 jobs:
   preview:
     name: Docs preview
-    runs-on: ubuntu-20.04
-    environment: CI
-    # Only run if PR comes from base repo
-    # Reason: forks cannot access secrets and this will always fail
-    if: github.event.pull_request.head.repo.full_name == github.repository
+    runs-on: ubuntu-latest
```

Comment:
Do we need latest? Otherwise, I think we should keep `ubuntu-20.04` since @jawnsy has suggested pinning these in the past IIRC.

Reply:
Not at all, I will change this.